### PR TITLE
Add "-device virtio-rng-pci" to the qemu invocation.

### PR DESCRIPTION
**Pull request by mkosola**

### CODE REVIEW

File: src/img/worker.py [hunk 1]
```diff
@@ -146,6 +146,7 @@ def __init__(self, vm_kernel, ssh_key=None, log_filename=None,
                     '-display', 'none', '-no-reboot',
                     '-daemonize', '-m', '1G',
                     '-smp', '2',
+                    '-device', virtio-rng-pci,
```
Comment (vigejolla) [inline src/img/worker.py:149]: shouldn't this be '-device', 'virtio-rng-pci' ?

Comment (mkosola) [reply src/img/worker.py:149]: yeah, will fix

**Review by vigejolla — APPROVED** [review]
LGTM

**Review by xfade — APPROVED** [review]
LGTM

**Review by dseight — APPROVED** [review]

**Pull request merged**